examples/horizontal_auto_scale.py

```diff
@@ -0,0 +1,295 @@
+
+        collection_formats = {}
+
+        path_params = {}
+        if 'name' in local_var_params:
+            path_params['name'] = local_var_params['name']  # noqa: E501
+        if 'namespace' in local_var_params:
+            path_params['namespace'] = local_var_params['namespace']  # noqa: E501
+
+        query_params = []
+        if 'pretty' in local_var_params and local_var_params['pretty'] is not None:  # noqa: E501
+            query_params.append(('pretty', local_var_params['pretty']))  # noqa: E501
+        if 'dry_run' in local_var_params and local_var_params['dry_run'] is not None:  # noqa: E501
+            query_params.append(('dryRun', local_var_params['dry_run']))  # noqa: E501
+        if 'field_manager' in local_var_params and local_var_params['field_manager'] is not None:  # noqa: E501
+            query_params.append(('fieldManager', local_var_params['field_manager']))  # noqa: E501
+        if 'force' in local_var_params and local_var_params['force'] is not None:  # noqa: E501
+            query_params.append(('force', local_var_params['force']))  # noqa: E501
+
+        header_params = {}
+
+        form_params = []
+        local_var_files = {}
+
+        body_params = None
+        if 'body' in local_var_params:
+            body_params = local_var_params['body']
+
+        # HTTP header `Accept`
+
+        header_params['Accept'] = self.api_client.select_header_accept(
+            ['application/json', 'application/yaml', 'application/vnd.kubernetes.protobuf'])  # noqa: E501
+
+        # HTTP header `Content-Type`
+
+        header_params['Content-Type'] = self.api_client.select_header_content_type(  # noqa: E501
+            ['application/json-patch+json', 'application/merge-patch+json', 'application/strategic-merge-patch+json', 'application/apply-patch+yaml'])  # noqa: E501
+
+        # Authentication setting
+        auth_settings = ['BearerToken']  # noqa: E501
+        print(local_var_params['_request_type'])
+
+        return self.api_client.call_api(
+            '/apis/autoscaling/v1/namespaces/{namespace}/horizontalpodautoscalers/{name}', local_var_params['_request_type'],  # noqa: E501
+            path_params,
+            query_params,
+            header_params,
+            body=body_params,
+            post_params=form_params,
+            files=local_var_files,
+            # response_type='V1Pod',  # noqa: E501
+            auth_settings=auth_settings,
+            async_req=local_var_params.get('async_req', False),
+            _return_http_data_only=local_var_params.get('_return_http_data_only'),  # noqa: E501
+            _preload_content=local_var_params.get('_preload_content', False),
+            _request_timeout=local_var_params.get('_request_timeout'),
+            collection_formats=collection_formats)
+
+
+def main():
+    kubeEx = KubeExample(kube_config_file="<kube config file>", namespace="<namespace>")  # noqa: E501
```

Comment:
<namespace> is not valid namespace name, i think. '<' or '>' is not allowed.

Reply:
Removing '<' or '>' relying on default KUBECONFIG var.